src/components/button/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button as SemanticButton} from 'semantic-ui-react';
+import './index.css';
+
+
+/**
+ * 
+ */
+
+
+function Button(props) {
+    const { className, content, size, textColor} = props;
+
+    return(
+        <SemanticButton className={`${className} custom-button`} content={content} size={size} style={{color: textColor}}/>
```

Comment:
the button could have an option to send a function maybe "onclick()" and the capacity to support "href"